frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -554,6 +594,50 @@ export function ContainerInfo(props: ContainerInfoProps) {
     );
   }
 
+  function handlePortForward(namespace?: string, podname?: string, containerPort?: number) {
+    if (!podname || !namespace || !containerPort) {
+      return;
+    }
+    fetch(
+      `${helpers.getAppUrl()}/startportforward?cluster=${cluster}&namespace=${namespace}&pod=${podname}&targetPort=${containerPort}`,
+      {
+        method: 'GET',
+        headers: new Headers({ Authorization: `Bearer ${localStorage.getItem('token')}` }),
+      }
+    )
```

Comment:
This should be abstracted into a function in apiProxy.

Reply:
fixed